gini/gini-vision-lib-ios
# Suggestions analysis screen

Analysis screen UI has been improved, changing not colors, but also showing some information when the file is a PDF and tips when the file is an image.

### How to test
Import a PDF and see that the title and page count is shown on the top. Also import or take a picture and see after 4 seconds that some tips appear on the bottom.

### Merging
Automatic.

### Ticket 
Issue #66
